x11-drivers/nvidia-drivers/files/units/nvidia.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=NVIDIA Configure Service
+Requires=network-online.target
```

Comment:
Maybe we can relax this to `Wants=` to still run in case the network-online target check fails but everything was downloaded already and we can still load the modules.

Reply:
Makes sense. Will update.